content/post/20211010-introducing-coalton.md

```diff
@@ -149,9 +150,9 @@ Let's walk through some of Coalton's headline features.
 
 In terms of its type system, Coalton's closest cousin is Haskell. Coalton's compiler isn't as advanced as the state-of-the-art Glasgow Haskell Compiler, but Coalton does have the following features:
 
-- Parametric polymorphism and type variables:
+- Parametric polymorphism and type variables (`Compose` is a built-in Coalton function, `length` is in the `list` library):
```

Comment:
lowercase function name compose, and I'd write this parenthetical as a comment within the code instead of in the text